--- OrigIpUtils.cpp	2015-07-03 20:33:30.325784264 +0000
+++ IpUtils.cpp	2015-07-03 20:33:56.184845516 +0000
@@ -176,7 +176,7 @@
     return Number(std::rand())/Number(RAND_MAX);
 #  else
     /* this is a workaround for gcc 4.8.3, for which the test for rand() that our old configure sets up does not work */
-#   if defined(__GNUC__) && __GNUC__ == 4 && __GNUC_MINOR__ == 8 && __GNUC_PATCHLEVEL__ == 3
+#   if defined(__GNUC__) && __GNUC__ == 4 && __GNUC_MINOR__ == 8 && __GNUC_PATCHLEVEL__ >= 2
       return Number(rand())/Number(RAND_MAX);
 #   else
 #    error "don't have function for random number generator"
@@ -197,7 +197,7 @@
 #  ifdef HAVE_STD__RAND
     std::srand(1);
 #  else
-#   if defined(__GNUC__) && __GNUC__ == 4 && __GNUC_MINOR__ == 8 && __GNUC_PATCHLEVEL__ == 3
+#   if defined(__GNUC__) && __GNUC__ == 4 && __GNUC_MINOR__ == 8 && __GNUC_PATCHLEVEL__ >= 2
     srand(1);
 #   else
 #    error "don't have function for random number generator"
